core/src/main/java/nl/nn/adapterframework/configuration/AdapterManager.java

```diff
@@ -178,34 +177,37 @@ public void start() {
 				adapter.startRunning();
 			}
 		}
-		state = BootState.STARTED;
+
+		updateState(BootState.STARTED);
 	}
 
 	/**
 	 * Stops all Adapters registered in this manager.
 	 */
 	@Override
 	public void stop() {
-		if(state != BootState.STARTED) {
+		if(!inState(BootState.STARTED)) {
+			log.warn("unable to stop ["+this+"] while in state ["+getState()+"]");
 			return;
 		}
+		updateState(BootState.STOPPING);
```

Comment:
Als er nu iets mis gaat in stop(), dan blijf je voor eeuwig in STOPPING en kom je daar nooit meer uit.
Stoppen en starten moet ook een soort reset zijn om uit problemen te kunnen komen.

Reply:
Klopt maar dat heb ik express zo gedaan. Dan krijgen we een mooie stacktrace. Het probleem dat jij beschrijft is op te lossen door de gehele configuratie te herladen